apps/nextra/pages/en/build/smart-contracts/why-move.mdx

```diff
@@ -4,90 +4,92 @@ title: "Why Move?"
 
 # Why Move?
 
-The Move programming language was originally created by a team of engineers at Facebook for the Diem Payment Network. 
-After Diem’s sunset, there are now two variants of the Move language: Move on Aptos, Sui Move. At Aptos, we believe 
-in building a strong developer community in Move and invite them to build upon the Move on Aptos stack and contribute 
-to the OSS. Unlike the other language variant, we strive to maintain backward compatibility.
+The Move programming language was originally created by a team of engineers at
+Facebook for the Diem Payment Network. Move is designed to be a platform-agnostic
+language to enable common libraries, tooling, and developer communities across
+diverse blockchains with vastly different data and execution models.
+At Aptos, we believe in building a strong developer community in Move and invite
+them to build upon the Move on Aptos stack and contribute to the open source software.
 
+Move was built upon the idea that other programming languages made it harder to
+provide guarantees at runtime, which is required for a trusted execution
+environment.
```

Comment:
I'm a bit confused by what you mean here. Some clarity or examples on the guarantees might help. Maybe something like:

```suggestion
Move was designed to provide safety and security guarantees at runtime, a novel idea not implemented by
other smart contract languages.
```

Probably want to add something else after

Reply:
I'm removing this sentence